website/content/docs/enterprise/license/product-usage-reporting.mdx

```diff
@@ -107,82 +107,85 @@ HashiCorp collects the following product usage metrics as part of the `metrics`
+| `vault.secret.engine.transit.count`                  | The total number of Transit secret engines in Vault.                                                       |
+| `vault.secret.engine.database.count`                 | The total number of Database secret engines in Vault.                                                      |
+| `vault.secret.engine.plugin.count`                   | The total number of custom plugin secret engines in Vault.                                                 |
+| `vault.secretsync.sources.count`                     | The total number of secret sources configured for secret sync.                                             |
+| `vault.secretsync.destinations.count`                | The total number of secret destinations configured for secret sync.                                        |
+| `vault.secretsync.destinations.aws-sm.count`         | The total number of AWS-SM secret destinations configured for secret sync.                                 |
+| `vault.secretsync.destinations.azure-kv.count`       | The total number of Azure-KV secret destinations configured for secret sync.                               |
+| `vault.secretsync.destinations.gh.count`             | The total number of GH secret destinations configured for secret sync.                                     |
+| `vault.secretsync.destinations.vault.count`          | The total number of Vault secret destinations configured for secret sync.                                  |
+| `vault.secretsync.destinations.vercel-project.count` | The total number of Vercel Project secret destinations configured for secret sync.                         |
+| `vault.secretsync.destinations.terraform.count`      | The total number of Terraform secret destinations configured for secret sync.                              |
+| `vault.secretsync.destinations.gitlab.count`         | The total number of GitLab secret destinations configured for secret sync.                                 |
+| `vault.secretsync.destinations.inmem.count`          | The total number of InMem secret destinations configured for secret sync.                                  |
+| `vault.pki.roles.count`                              | The total roles in all PKI mounts across all namespaces.                                                   |
+| `vault.pki.issuers.count`                            | The total issuers from all PKI mounts across all namespaces.                                               |
+| `vault.identity.case_sensitive_mode`                 | Whether or not the cluster is using case-sensitive identity name matching caused by historical duplicates. |
+| `vault.identity.force_deduplication_activated`       | Whether or not the cluster has had the force_identity_deduplication activation flag activated.             |
```

Comment:
Only these two lines actually changed! I tried to keep the whitespace aligned throughout the table but sadly that makes the diff look huge.